client/src/ui/components/Cell.tsx

```diff
@@ -15,17 +15,29 @@ const pickColor = (gridValue: number): string => {
 };
 
 const Cell = (props: Props) => {
-  const { x, y, gridValue } = props;
+  const { gridValue, onCellClick } = props;
   const { classes, cx, css } = useStyles();
 
+  //TODO figure out a better way to display destroyed ship
+
   return (
     <Grid item md={1.2}>
-      <Box
-        className={cx(css({ color: pickColor(gridValue) }), classes.cell)}
-        onClick={() => {}}
-      >
-        X
-      </Box>
+      {gridValue === 3 ? (
+        <Box className={cx(css({ color: "red" }), classes.cell)}>
+          <GridViewSharp sx={{ fontSize: "30px" }} />
+        </Box>
+      ) : (
+        <Box
+          className={cx(css({ color: pickColor(gridValue) }), classes.cell)}
+          onClick={() => {
+            gridValue === 0
+              ? onCellClick()
+              : console.log("Can't shoot at the same place twice");
+          }}
+        >
+          X
```

Comment:
<Box
          className={cx(css({ color: pickColor(gridValue) }), classes.cell)}
          onClick={() => {
            gridValue === 0
              ? onCellClick()
              : console.log("Can't shoot at the same place twice");
          }}
        >
          {gridValue === 3 ? <GridViewSharp sx={{ fontSize: "30px" }} /> : X}
        </Box>